src/main/java/ladder/view/OutputView.java

```diff
@@ -0,0 +1,29 @@
+package ladder.view;
+
+import java.util.List;
+
+public class OutputView {
+    public void printNames(List<String> names) {
+        System.out.println("실행결과");
+        System.out.println();
+
+        for(String name : names) {
+            System.out.print(String.format("%5s ", name));
+        }
```

Comment:
```suggestion
        for (String name : names) {
            System.out.print(String.format("%5s ", name));
        }
```
사소하지만 컨벤션을 준수하면 좋겠네요
나머지 코드도 확인 부탁드립니다!